### ISSUE TITLE
What are some good places to SSH into to have access to a good terminal?

### ISSUE THREAD

**Issue by julkhami** [blinksh/blink#1288]
**Describe your problem**
I want to use a full terminal on my phone. I need somewhere to SSH to. So far I have found that even home computers aren't great for this because you have to find a way to wake them if they have fallen asleep. Sometimes you have to relaunch the ssh server as well. I have done some research and I haven't found any significantly good terminals for iPhone like Termux on Android. The only idea I have is to get a Digital Ocean account so hopefully there I can get a fast machine to ssh into.

**What you have tried and your configuration**

I've tried Termux, SSHing into Termux, and iSH primarily.

**Expected behavior**
A high quality, fast, full featured terminal on iOS.

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6] iPhone SE 2
 - OS: [e.g. iOS8.1] iOS 15

**Comment by git-eri** [comment]
Well you have some options:
- Home Server
- Raspberry Pi
- VPS

I personally use a Server at home with some virtual machines and i mainly use blink to manage them when i'm on the road. But just for SSH this can get expensive for the hardware and electricity cost.

That's why i would recommend a Raspberry Pi: It's inexpensive, doesn't use much power and the Pi 4 is pretty fast. You only need to setup VPN to your home and you are good to go.

A VPS would be my last option because you are paying monthly and i like to have access to my own hardware but that's what i preferr.

tl;dr: try the Raspberry Pi 4

**Comment by julkhami** [comment]
Thanks very much.

I actually have a Pi 4 but I actually don’t have a computer to flash the OS
and my WiFi options at home aren’t great so it’s not the most convenient
option for me.

What is a high quality service for subscribing to a super fast remote
computer to VNC or ssh into? Digital Ocean?

Thanks very much.

**Comment by git-eri** [comment]
I haven't worked with DigitalOcean yet, but on first sight it doesn't look like that they have a service fitting your usecase.
I think Linode would rather fit you better.
But as far as i know, both have a program where you have around 100$ free credits for trying out their services. I would suggest to try them out, both look good!
Have fun!
